### ISSUE TITLE
The window is no longer responding

### ISSUE THREAD

**Issue by sekhuat** [Microsoft/vscode#39227]
I am building a RESTApi application using Node JS with VSCode. All the while everything works fine, however, these two days I keep encounter the error "The window is no longer responding" whenever I try to debug the Node JS application. The error didn't happen immediately after I press the F5 key. It only happened after a couple of minutes running the Node JS application. 

![image](https://user-images.githubusercontent.com/13288516/33301206-728e52b0-d42f-11e7-946f-5ad27e72445c.png)

After I click on the "Close" button I found that the node process still running and the Node JS application still working.

![image](https://user-images.githubusercontent.com/13288516/33301232-a3e73b06-d42f-11e7-9f2b-7bec35a9b398.png)

If I try to run the Node JS application via command line there is no issue or error. Therefore I am sure that is not my application that causes the VSCode to be non-responsive.

- VSCode Version: 1.18.1
- OS Version: macOS High Sierra 10.13.1
- Node Version: 8.5.0

Reproduces without extensions: No

**Comment by weinand** [comment]
You said:
> Reproduces without extensions: No
Does this mean that you are not seeing the "The window is no longer responding" alert if you are running VS Code without extensions?

**Comment by sekhuat** [comment]
I didn't try to disable extension, basically for this Node JS project I didn't use any extension.

**Comment by weinand** [comment]
That means the correct answer to the extensions question would have been: "yes, it reproduces without extensions".

**Comment by sekhuat** [comment]
Yes.... any idea what cause the issue?

**Comment by weinand** [comment]
@sekhuat so you are seeing this issue even when you are not hitting any breakpoints or stepping through the code, right?

**Comment by sekhuat** [comment]
Yes....  it will just become non-responsive after a while after F5 pressed. No breakpoint is set at all.

**Comment by weinand** [comment]
Do you experience the same when use the "Start without debugging" (Control + F5) action to start your program?

@roblourens any idea?

**Comment by sekhuat** [comment]
I just tried "Start without debugging" but the same thing still happened.

**Comment by roblourens** [comment]
I have no idea how this is possible.

Have you tried with any other Node projects? Can you try debugging a very simple Node script, like

```js
setInterval(() => console.log('test'), 1000)
```

and tell whether the same thing happens?

Can you post your launch.json?

**Comment by sekhuat** [comment]
Below is my launch.json

`{
  // Use IntelliSense to learn about possible attributes.
  // Hover to view descriptions of existing attributes.
  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [

    {
      "type": "node",
      "request": "launch",
      "name": "Launch Program",
      "program": "${workspaceFolder}/bin/www"
    }
  ]
}`

I had tried the `setInterval(() => console.log('test'), 1000)` and it is working fine without hanging the VS Code. 

I also found out that if I run my Node JS app, which is a Rest API when there is no request to the API the VSCode won't hang. It hangs only after a few requests from the client (mobile app). Even the VSCode hang, the API still able to receive request and response back to the client.

**Comment by roblourens** [comment]
Can you share the app with us?

**Comment by sekhuat** [comment]
Sorry, I can't share the app as it is a commercial app for my client.

**Comment by roblourens** [comment]
A simplified repro or any other details would be very helpful.

**Comment by sekhuat** [comment]
I tried to create a simplified repro but can't simulate the same situation.

Below is the log from VSCode running in verbose mode via command line which the same situation happened. Hope this will provide some info for you guys.

[main 12:15:29] App#activate
[2301:1203/121532.577501:INFO:CONSOLE(9)] "%c[Extension Host] %cMessage Debugging with inspector protocol because Node.js {0} was detected. didn't get externalized correctly. %c(at Object.localize (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/ms-vscode.node-debug/node_modules/vscode-nls/lib/main.js:56:25))", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js (9)
[main 12:20:33] App#activate
[main 12:23:33] [VS Code]: detected unresponsive
[main 12:24:13] [VS Code]: detected unresponsive

**Comment by isidorn** [comment]
The debug console is using the tree as an implemenation details which has a bunch of flows. Including performance implications when there is a lot of content in the debug console.
Most of these are captured here https://github.com/Microsoft/vscode/issues/11462

**Comment by sekhuat** [comment]
Is there a way to configure VSCode to show the Debug console as just plain text without tree view?

**Comment by roblourens** [comment]
`"console": "integratedTerminal"` may work better?

**Comment by sekhuat** [comment]
Just tried `"console": "integratedTerminal"` and it seems working fine 👍

**Comment by roblourens** [comment]
@isidorn do you expect the console perf to be bad enough that it can lock up vscode entirely?

**Comment by isidorn** [comment]
@roblourens well the same way that the explorer can lock the tree if you have a program running in the background that is constantly adding files to the root folder.
We are already doing some debouncing to not refresh the tree too much. However if somebody can provide this corner case where the vscode gets blocked I could investigate if something can be done to improve this. Apart from this I wanted to put a limit on the number of elements to 10000 but I see we have already done that with exactly the number I wanted hehe

I tried the worst case scenario (since my debounce is 500ms) and I could not block vscode on my machine.
```javascript
setInterval(() => {
  console.log(console);
}, 502)
```

**Comment by sekhuat** [comment]
You might want to try to log the sample data into the Debug Console multiple time to see if it cause the VSCode to hang.

`{ id: '51C75017-1C82-4375-B50B-5F857AE207EE',
  linkId: '82BA0AA7-3CBE-421D-BE3B-FC09B501551A',
  linkType: 'MEMBER',
  iss: 'net.justnice.abssmobile.server',
  exp: 1513164664,
  iat: 1513163464 }
resultData: {"success":true,"code":"SUCCESS","message":"","data":"U2FsdGVkX19wQuzoqrq/v3nlVOrMIEOZ1nhdxqUCUYqSSVX3roa3ijWyKoLSmupYDdxT0+veT2a8sVvBMudMCA+cI2o3fXaTLuygEDO4WAtcjwecU9BScJNanQMHgfWT6Ynla5O5NYfvXwWR8V+4Fxjs3kGeISIO9+1fynATeaNQ37kq8/HSs38UyuTszE7E7ZPtfVpTmmF2Vwdh9pmqtpoR4AXjNFQIrnsnh+qm64kBD8os2U7OUk4RRzQcS56qy4rq8JQrDcvhC9yU7VSk9ZNEQh+GVbBLj2s/qkYYsBAxLtGVvXN0uV2fHLZ2sJfKh9GgWm9m6YpjV11SyeOhRdbkfH/HG3cUM/wdHzg9ynlwIm4lMT+mCtlwXPEJYQel17QZXeC/31KRcU2OpYIR5EhF32HT+RYHpabdIFl8U+74d9eHrBgpgRLya+a2nRmMYR9AvzQAVbCgnNtXAjyS+SEy0qEtIOk7/NpNoVVYTUgBRVmNJ9sQL6mtPkfqPmhWO8IZfGC70lof0Yd5W9bHQNh0ptE40dXeEVDvJqkKlpWGfU7YEaP3HkYzEnFKWlmgN0rvdgzok9quOLkgQ68vkFJYh5IDygCXh97dPN5kg48NzjmyUApRvLwoUab/YzIj+5sSMGNUZ6vf865+cfmtopFi//XvcDftGVwfBAwbvtV/4DwLXwl4ER3nId+bgkJWL2q9u/4kqooP50GZ7FiUNxTIT+dhZ5RukRmXU9uMdAGlIIcoyRQgV3REXivjtb5uqpFJPAUo8NtEQhu6o/fTjWs6ibAyQxrPsJKSO0jnyDKpT/E9hsw5vpfKXtbj40LdliB84JzoCsh/qk0L2+I4NlVDT0Wk4YkqH7RAT10j/RI72Vtte/J+G37GNqjPoLhJuIPeQ/Fzd/AKhLbnI/xi1mJSdTIc2VK86OUWvBwiAK658e/m7b0gPJJ5pBhXgfhH4SulNSuGy2oLjyhDcHURBCVYmrIrnKiWgH7yQLZz7j3zBSsFl+suPDR/6SdamgDKpjgyKIvBA5H9lpxxid38W6x9/f5AaMjuY+TDnDFo3tAyUZv912v11eBit4SgG45dwH1CA/azwnL8lEZcc9vdHfEazcI5nAyzuJMEzYQ3kfYJfpBdVC6KeAwWWhPsoDvxfnWwBQimZ0h+B0Bw9dXzbNCELJ1smlEmankV3M04PDOlaMqgjJdtRLNTvdqU46szTBKM7MCJk+jLTyuMspMZMpbXBiT+zWWV0RYdRvnWuhghUPjor5QEuh/X9Vt48edRjqymOxnMSzlz9bK1eu2vK9kMoVT2Bxrn/fBLqwXim+03cHy2arGFeYQEgntcKUEKjby+gDZVn0jFIT/dLXpG7Qjb5VWOg/aozcKfJm3sp9esb4rOv9aEB3ZARTnzxIaVTsixbzmuG53vzsJSM/tjF73l1Yb8p/K3WBxo2OKIaD5VSRZ+W9FH9PbIQvB6pZ7QPT3UHkv/uOmzbwD0RylWK24IdfEXm4zGGL3JYwx/q5WJbvmXyNyDXIeAd4boZf+bR8jAlfShTYqHsRDlVNYbTGo4fdc693dYD/76NeWV2znm/qAi3iVJE1z8VnKjdWCreKmgU3BKwbVtcbF5KbUNKiCSF5VDyo4PpuQhx8lhc895GWZcR5goR2zOCz3gDTUvG43n112iCgrRloT9d7zTWOug1a1iyasDss5w5/tmRj1NNyVTOpXvFFZ28cO7kZ30WT00ENh67NtTQVRD/N7CkEzizxCLiHq7c18Th+qtHnxzpaIrov6IbaXQ5v6GKNNZyGch5anE5OL4NsQUQVNglq+btFmB+Nk6K4t/s0vVz0P8z+kuH94tB/Vc9fMY5hmYq79nhyaDE5rz7k/3GuRhWv8URdJ5kSiiEL4q8U2EiarS0m3D8VjgggydiOC2SMKSugC0+zZN5UMF7+etvD9WH/B4Fh3Idc3ySjwuZxzqUKaHAxEQRJGzuuU+zBJzZ4c34tq5RB/fMMKI1KSHP7kUULYjYi1q8/2Wuj/Vowo7Tbeuu4HXqtUz7lWydKYAdywVDRQfDteGncGml9+kPNCzRgVlq5uxjTAV3daMY3URY8HEJ+i66SGbewjcGVOtOYZRqzfKBXK/mceoTxVAU6kvSWf+IfijmtTzsBJMKtY+lb1do7Dy9sI3yog9Sm5e0Y79FLaEB+WSUuoI3c2Atft01SWw55hTPLR8DFvMHBd6SKViSj8C1KG/1L65jDNAFd6EzkgrWSDm0ayXqmCotDKFx8N9BdR9NulG/tQoW3fNSdoabzyMsdT4mHvksNWvczyZG/0SVMy7x466ttIlcuFrVyXMbwNpf/3vVggV2tGcl2WNgQ7g5oj/dk3CDoYBdK+umob6+IZDJ9um5ZftiXER5f6bjyPo4Wsx5NKPZ+Fu+AvJO3x4Ng2d4I7NlCYgL8UppVO8NnW9/SpYt6y76OE3Uxge3FoXZLagf8gjioWOD//8hN850YwsqT3XpNvnp/ea4Ae4vsltDg9jtG0L+P71hdOj85VMUhK0axVR2JavD5f78I84+NFjBjwH9r5fa8nzg+1e46W9tvwwm/GB3gbnmn6bxDG5+oz4G7Oe9kpCbrvJJ4qohroyyDGeGPyTTRZZjHbk4TcSxDLGRxGWJawOpuF3kOsFwuQFwWQxWzM98GOtpoS95Txo+qo8lIeDlWdZ7NVlE4taTYd67HZKRNJ6KYgtvBmS6tJRiVnL+Cq5hqT1oA06yFz2CoUuPnYUEufOTWDCWDg1GMW10C8fcKicKEauF9sz2PzlZHh/KBt5UxVUECJR9v7D1iMTHnzH+veXj94CIkmT/b2ukFDVw94jL9PWHKnXCH7T8grZXTQbT7HT2uL9yJ+/iT6M7BFYJE/FHQKpu5htGd4ZjXeuON1hN5MH2KVapuiQQ73cwkh547Ae7e81uimt5R0S+WR6A1jPF7GibHvA6sdZMfYo2oer7uhyMsOuy465g1gz9yb0VgAwOO6I+qXXIlZrcqFoVzWRQX032UV92V+Y1yveKX9COAQ4jQMh7BELRtQstR55ofkO62mhAYv205ehRbSTmZ7xBKW7WuXcrsid0fCdJyoDH0FkcYOpZ/qz9c6PsTnVrKLZn0qaKoN82WLC7U3RphXWc3aZtrzL5y2qaSzo2vLy6nCMTfK+YjvDv8MgFYjJGsQPrnju5pocI3JiJwCoaDjpyyCaklutUNK+nf4DVM8JQa2BYvNqNhOnJigWO9oZrTfVaZ/m2/m1+rc1SyMb4sN9kX6GEgoBgWZ+UI0qjK33rsC4Ao7fIJ7wD1rJ/iyK7prl1m9po4apRSbFkIvRlmLX3TyZASuLSZUe5vot2toMfF86wXg852sA79OTR4WqkRMX75S1tP9vffvoGtFnSqYF1Z63vVMMDfZlIrb5OKRJVlRij+UuYC9Jg8KmMN92IE7ccpPpcdlJqueS0wJYba0fU9da0AJQLDJsF8JDSoo6C0n0tz8EKHhxIUoqVHt7LAFgmogiXKrppYifY18F6RFuief1WumUs2XKJa+ODxCh7Wcyd5R+TrHwaV8MM+BTBvmmjLVuhr9O3R3MIgmXz4Gzb/lKRGO+vh0WlC9zkKo4QtuBm/0hpSP1+88WYJacyGe/tK4TbwSFu8TPWREzuWofvKYgUAyGsyCZy0jusZQaNT2TcgoFkes89dZntuMPGuEpiFuTO607TOCjvrRelTgLFXBAeuaejFsTRy3KTYheZnnSYgN1l5YUt+bD7xDv4zCH8WgCk0D/TrU/+ksqn9uvVQGKOi/GXOzF+OjcqZQWEATMo9Y0rHLyJMr5Plpzb7DHXez4+y/46/BqttZJUmzusJ4SYaS48GyxCC1SqUFrlI5RiC+XNpYwNyNdXzh6x2sluSNf7dlHjCxNO12gPseKr+JVoqUmq5yLA82mBgqz0Lh6GZo/X3+WNxF23VEDk4tW/YBYaszyILFdv/sUGBm89ePyaT3xqc2kwgLGZO3yTudZ3CPEgyrOR93aJQUDIDT/GUasnr5WyoMILfhfy/UbpF54XVRdLAuyNPbd3ckSAedcViZWfgPls4UBqKwPpB52X9VecPKJ3yHZ1/PNFrzGJhCqoK3UKYavOd7eQLzJmVHVreLTyaRRdJIxiX2CCVsTHKM8z4OHGqbsEbzAKwkxJiuPEhwG0I9SAEbvBEdIC0wl9PmM3Ntrri9CQGDIUZK87jYfN5xX5IQ/a5huX5KIttNfMpahPRciYcRViZJENPLtBDsf4+t+Ss/E75EKJ6ECyNKkAWFyoCmUwJjAb9Xw85uBxurKrH3qU0ON+61uTSbUBztrucnvTvV+0tHtRDVJysUYwK4nx28O/mb+whKgavxN5l3pGscpIpLqmV/UA6naxf/AKz+RsDSGHfmI/azUaxTJFJ+K80IJNgfV8ItmZdUricUkuQ/FtlBbJeNKLVP0yvTM1ZcA80Gi2Y5eNJ8zO5nu5svUx5NEHIV0thPn1GbZYXDwTe76lQC5w0Na2OqaTJucTiLJlFmBoXcvbsf6c3GO8phmWio7n4Z1C7afJizV5bSt1ZTy+p/hFAtdZjyNkKUoHuhEZpF3Aayc4G+bNHFhm18f4eoELRUhBHvlmbzdzRBABSF/6RJg6mMouTJzPxrR23ABIN35p8SiF9zCcCDOaF+Vv0yu9EQ3AiagQibN+EutWStfHc9JER8rViifDD1GFvuV1RHDCjQGagEy1ObF7n5hQkgGbh1TQdI0phFpGoN8+WZK0eL+UhbN56IYOz4lczGU/3RJKiukhWbbr5l90GIx81+ve1gpU2Yc8xmIBIUYHdp5H9FHJt1JKBtYMMoi7jzdZNcihB4/uVkFXORKAFpBevoSk9NK68mYkfNaXuOHic/3dVkX3VHKLUxIGhw7hKITWA7pPCJB3B4DEyPxgdPmnzg+lUVE2g76IR+80+UH+7GSIx/YqdbysMtxBGqfCjE0YcWauO0RXjkxgTqw49CPsYWYVh/kOroffkzFJAR2mSP04rgNa7wwgrVcvP7NQx2xbJ4Yarpv9D18X7HjrxxdMs2oBbneeGbWnkElKRt5ZBntozxhkj/e4NXYJJyDSIZX96GV7ATABEXQijzvzOLbmY7qrt71QTJkXFTWIe/Wu+9xF64YVum0C8l9ib5V5e2mPac+W9q0XyiWBYK22tS2Fv8tCmruCJS0wwhgEO+cExfdXf36gbgAur6ogwZJtFMGN/RulF5XEumWxNxz1nvbrLJfL3ZeAGtpRh33I5GquaJaR/m/TNcgteFykoGJ0kTs57Nulfo++Xv1kEv4iOW9KOtc0IUImfd432IU+tOnjORgj4ooJz88krFHa7SQOLXI6fUEknxoSUp04m789IsH2d9PNNtMCB20HF3tSJEBKFHYA03B6BeHTj4zGJ6ChetyWlzOgeMB0T6yag09pfg4pArtAA/IA92ck/K5TnqvLS0/BPSvOd1CZ9sMpp0j4sFj7oWyLd6aZjmnQtQ2Uj/09muKA18ywZh7Pnc9FL5F1EaEQ5PEV7S+43dxx1Qotkzm7x3Ak8X/xTHQRBUQAthX8ztkH6JdDfKyS7eIo+hZ+yDxdJ87EzYNcCUhBzG4GGg5NI8gtSr0nPk9nCNHjakvzIkL+pWcPmIosQu6tMi8QM6JvSiKPdUvWsn7wMh72G5OLrzWuCa8HyUtRF/DDa1B6BkGoQkaBM+qn9DxjErF3w7+NXnBGPJCZiBkXLdw6GNRLuCTwntksKUSLz8/FD0gxm8cXOba8UU/nP9+G74ZRAfASzqOOX4nDGf4tiV8Ur8GSkKMLGyJIZXicCo3NcSalGRl5TCSqPPVREcap2ZdM4AwHxmHKXJX0zrHH30G6ctZ9nojeQneu9yi5d+ypOdVE/n1fhjGFlI4QWMapon5xTjD6N/CCEl79xBZaMpCiuKCcrCyDqyzsJNHhBjESc+k8g5MfaJQpoD2Op2jwSxCPfhHPhb5XxAWHcpRgO0JZS/YbS6VMguVQOoq2kBs69Aq8JAQ5ViV6NO7k9hfSSok//TBjjj6gBHrXsgylTDVI17BEXzlCgRfiYadP+eIyygEy5k+XHVf1BmFnCEcQNRSNzTG42Ihu6SeISUb3PTPD53wYeG2jxgEmGVCPdA+D2AvvD2sRfGv+WBj0Zxp1TU8GocGKdhGygSPMENic3uxfuMm/mBYOpKAH2kvoz5UnvEfhFi/c8YZhP13oXNNLJq5NkWbaJD6vvRyj1cpVoJRgkcfdqF6ttgFRZuP8IYGAH28UuJZeeVAcSRfRHtWi0no0bl8K/QemK4NuAOXvw8S6amStObwgrnAl+6kN1fe+dc1/uT6Kw1C5s5jEabY7I/dhL+t964PhmOBEownSzZ3py4GKc1TLqBKuFxwxc5MPrva6Mjh2f7rG2ZHeq9HYfBOEcIVM+VEiFYIfVCxdFhOa7Gia4cb9QA6+ppHkEDBjpmBuNQBK/jzjrKWWrTV3sZ99gUmuOmtEVPXCwAAA3PPtsksFC5L6YI0LLpJPw06z1UCbeM3ZVQqaZ/Avb6udHZ8wphvqiPi6ccC5YIfXd1UYlpR4lcfSU9w00NLYX1RMEVib8g53rpIx0fH7sN0x+9uQYPnH+8xRTHIEf3wEbWvLmr1Q4cZn14P5f4bk/NArWB0YJZSmqtw7zH8N6JSImz4n/VDA4RHGZfM6Lz5dpYNx3rPmwvweWirnQt4MUoUGR7V/pdxeOr91Uy2Su7NJvKdMzdJA+R4IFx9bKiXJQCsQw4B5+Q3akNbNPpXe8I6WWHadK8m4qp2ab/8S2tzGMv1rOcxbHkqf4nwq+NkjvdFKvH5JbZg0qFDssYsaDOS72iu2GbrmvtBg8AYbLh2HcDTzHDtH/GQWsuJT//lWR5hMA7eGrfXPYFZNT84/NJ6JnC9JtGWAgvY+l7KD9pgcapkKTQfFjGemIZ6H8VQx3pIo1f4KcrKF3rfhJX5Qo3ZvuCxJENg0J2L4NxYyXPBIGRqtkdNTwr/BNm9VmUjdwCGMwlcqnAh3ENvHVoaDKm7CNRJcsdnBIaOlCFlMtA4q2vgQsLylGIEN++pfrgYJwpO4pCzT3Yg7YomhoX2B4XfQv2XRYYl8/+JY+jwfQvHeKMG0C+bscWZ0cwrrfobBOtNAlzpDTi/KVaDNNKj8bYm53rSvm26avpc4vDnChjhul9gF0SA4n0QGhW8oGjK+S/NuOeoJRZGbvxS+sBU4y508BTGsdFljkEcThWkmqMEcxth0qUcRXx6cnFg8eCoi+vWlJDlGeRl/Ss1vKuT6K+aNRpEZ8mtJ29cFpWqDXtVquMXMXHIsLOb0dsP8q2IXI9wWLtXfsuD4p7yjYgE1leSlvRFQz8XdN/DW8oIeUrjzaIOXg4OqG3hSsiXB+VrnoCKDi94jacJI54XCOC7erdXqvAQSyaCwhn2XgR9sBTTiMLu9UqnpcUqnX7CL2qr0C0ok0VuDBlMS0+qvGss0bCFkgiiSBOUqvYRpKAOkBU44UaG7YGCEpfiLmPTyx5pYx30nJ1IJIcrJZgtmaf7AHpkA7Xs498X3ywrGTTkmb786MPTW/I5m3g11CerAZCx4bbihHPrsg7RGZQI6JODPkPne3i5k8OWLGoF9E5Pkm2mOA+QEPZefX5gpP6Sraf9eR8eJsdh8cF/NRkIM8R3jjZ4WeLToR2ier3b8DPNjG45KUYPimEr5omTIwgRjpxgMcCN77aOOdrRYgEpBc/QBP86G3OBaHaT7664X2Kw+T4t7jaTqJImImqXO6bD9uaOX20iETEag2dfJVoqs3lH90+RcFjYOmSVmoHM5lT1GQUuDcZfjaP85FXCup0C3a77Jo4w7gx0SXDqu0TQRLT/yE32dIFcNeDPXi6YUOKV0WXAr3rd6aRFZGUsbnBpCF9wME/8NrXudOwwQXeQWcYjatE2CPWycw5HPPqQYO/JHYDX8bxvbdXBzYLjcDNRHT7xEaZRFwy/XUyRrklpr99jPiRtwye8h3TDJ6bNPWSOmnKT1MgDs/d67Ux5v+VDtQbmcGLbcFpDEwiRrFt73iaX75NH69VKbhvghRrNSfFKFeHOxFPDY8QDOYanmweoBZxSWV/P2xE1EnW77/AB1x+ufYcaScHPeJKoyiuS5ipKxxRO1RBUuJTQddRCPVDLMXFkmvCfYDK0dCll89Z98Zl4MlrM+5eh2LTvP9tBQs9LJv51VRuYoH+45z/VJuNBx4uyG17JfP38EgoeWgxvJqIX8vycWNZ6qX8xoUUqnE6ob+RctuX75CUM3I1grvmJxgTjt3+ZB/T/p5EN/77zZsqnnzz7ZOz4EMOayvQOcicYLK7zQd6sc2OXMOrNuHeYgF/fYwQkIP3lt8apl2EFkuFutOgAaZWwB5dQDmz8Ytx8OaZL5qmz2lpNrLiTPgf4yjvozgJHd1pc4haiUtDHqBJFH3w4GCVU8BMUDsM9aOnkNnTFFrseLnzJ8LLOXMkSfj5VjBO3QtKucniFcxVg8ALD9Iblh+3vuUQD4IJRhYlitTIlq0HP5ldg5EX9gYT1LSHbJ5Ei7RtkbkGdcpZAGBuTamB+0ZiXK4kCsSrxLlOWQrdHj+ru+2kGhiRK0YXuvSHS9ZkdcekCqG7Txi3eJBhcPLNWgQhWtZxxmKOmy0g/JQqlxX40Mxvgu7KSouGdJlZlYIwellztB94r7z3SURi4pmketvD8U3o2eK6noC6QI9VQQHcfb5R0+dWpiqekKJadWfQqBxnUyBRP114HNnYfJz7oezam8g3qyBlthNCgJicKsloyAVzyEV1S9e3PYZ3JCTn09VaPKuB1XLnx2Z4sBJ6s/GqI+7gbfZDBY8zaorRFEoqthohtQxjmPF+MZWf/XQkDWfenpfdVqc9bAryUobCLkj/gVJAJ1z4J8zR+O5BClWw+F36MuqseKpJCn2JvYK57bMlEHUs01i+qbhpbF/WQcr283PbNLxGgxD+j6ETmmULHaocRq7m1/7o5ki433fqyhlwXoC9ujYR0N+RmMlh5Ft7vq0y+BvYSwHEpmXE8Odq13LXph4TnVGtcKpSTcWVi6zTQKIK9YChzxJrmpU1zkSMvvWTzDfnGnlllPhcdHPbkBcR5/073u81VisVS3oc7ztnTc1IABR4hRtkVEgMr7OA1voNy9/Z6qvbPyunK6w+Wh3m1yAPgA5Xad1gUvT4m5NruKU9K6TwJTJqk1I7KeN210gc0L5QghdTjOO93z/bCBk4xdvTxdgt9jemCONahM0d2leb74MQJ4wi5XK+Mj+Kkd2MuVkQo0J68N9gtt+k3jWrGUe9PyYg2PgXEevUjaWxfm9E/PPg1cyHA3+JoIKLNc34BUFBwsTh9jw2rPClXpeW+JFsONCb+nmhkHX0uU/UyHwdBtxtg1cVxi9JQGCPSZyMyujMX/jxKUcFkQiH2/Y4Df3g+FKKau8wjfDgj75jf2UuUmsJPlY5rZX6c8WpJ/gdeneY2bhNXVG6Y7zXyik0DM0SL9u//yUOEtfp2sPsnOGrCnyKRR4fcQbLAsvzyPQTzdRHtstqyWDZ0m7zWuf5lZ6ms+BPVroDsRtNRYKRqO3GWlECNKIgegKTzPKdtzYsuOUwirpZYSX1/Ifp6vDLYpoUN0wlT2TEG6qLma8lNVLQ5Y1nwERJnF18wVcClPLfPInmh3rYMtDbjGt1ccO8wGJ6CqJN86N9e5J3zlQtyA2SEpp3G9tz6ExSzSiDpYVfLkFNN3h6X8QYYfI8xu++a1rbPQwG1Onu0VYL2aNNG+D4J2gevLfYnnxBNmxnVsxuPjdK95G6vbrNyk93ZZWqqqYcYzOFGj+njUhOgU2TR09ug75PlltONLATumxb2NCa4yZcVZ+Ijqbkynjsjgmp5TQw2IooajSEUrXDpo9eTSWtD3SDxu1e0f7ww/Fc1/K6txWGFB5xCTgGPHrRB2RuHwBi2DG4oQGGWo6BaniHh+hlXvs80Zazxt66J0jkXkUakEqEHS+1mUnFLHiYwyDf9Lm9L9asS3br7Ip8nNp3KcdJdlwFcW3gIwhDBqFmIQObOdYX0WjLDQKDGwx+n9mq4/HV64dkYpWs7W4hLFETycR/ov5v5PRu2ECKXJcEx2HLXJcXGajE5EQ0wGwHGM5Jj7SEUukN0m/llaLli3vWfN/tx6lWpRlQ/oKGP6mAQHE2A21Rp/Z/oSr5YXFql9moz135H++SxK6wTzLCCDMCmoAGYReQU/rr5w693p87YB5eWI1r8s3FZpzReZjHluJJ0bFmOdE5omzMkFWbwo5TK+HtEtMmxUShtnU0ofhbwP9ukTiX7XLjxgYN39f8wtDup6saklVeTAW3xgr7VgIvt0drtR5DO0G0m2w35DgYGdj5SriODRxinTEoiqYLRhAEhWjudi8XDCiZSlZ6fuA6RqrcV1jyxWgGoYw+E95vhQ72t66rcm5uEYcYb12O1nujp3H5y8KJiWgtzuxJgBWd9mMHFI6iNTOzwP75ZOz5IHhCuDQimihtSHkUJqkhXQyGqMyfZRCp+AHeGXRH01osjjLm4om8o5qnEH6+app/OCUbuH45NxJbFKRLavf6OBcpWMXMLByGTdkvEAMrzrwsBIX5cvQrr4qZYjY4zXnTtOoXJ8S5gXZ+rRFHRJqaj4tUjdO8QMXNRIx+ehRYjmFWkmrbgp8o2hoDkQF3/Fu1dzONfcy/QSBCBgClAzG42v1PjQ5W+IsnYV4EPp6ajeGjKLtOBeN3qSJsJLdCeQwM8iLPveemf3uXt2oVyEqGuTRLqz0BixSjgUtaO2uXCGoG3Ot8lZEF74QSVjk9qIEaV/DoyX9s8zoRBL2HbZsZ39s/RhcrfEeZiJOOl5N2eINgvk+IPLGc5TEGI8j0G2j+5WmTTgnIGC4m+pr4XSyncgADizJmHVEYl54wxfdn4cdvNY1n2+c/3/7OdnEWp0NoZVcbdLBmdf65gYBpDsDgWzAAGA6W6J10doIwqU9HmeC40pZPalehSaNfXGjWdNTFL/p/YyHTALeVGMZfmDCKV9AGCCAUhc6KN8o0F1dJNTJ34Ld/jj10huPZCn06RUu0ElNaLWLn8wusVq0N6W0SYx4AAouxLUOMDM5eIOeIaQRyxZlP+aETbf8GrFBqjrjk42meqxtAAQFj1sY79e3yWjMGnu6QYySybbmjMYc0/uwerWtgikLrtwb0e7zoy19sHKNNXLeVIUoMXAln2/mlw9Ts/YA6NFw8VUV1SqaZ59MblZ2spZxLRXhcxHgm+7QTVzG9nv0CTIzQrtVhSBdhzvhxgTEiHWfCmJS87C8HotCTLOw1jklMLnmDUw49IPR2h3gFNF4jpcV5tYTTG4YQfJovDzOCoSdLrCO8oxkhTJV+kNYzUMVd73uU7UtfzwW6+CB8ij75fz22nrM/wB7cnNqlW5DXBp3VpkBK2TSAYonCVl+JTcbMHzJjUD7oJ0lTBCHBcfx6KcatrrshpD3k6aPYXnm7we3TC2tHS4NNY34pzL+/0Fg88+5jVf5eydcuSkTSy+Csrgs1PWRpidjB6Umij6FCFaVjNoH26VG6efGIZZg8V3Fi72XDYJDS42C2ONRUib9ngdd4ReLDLB0+DCkzpImJeRjxLLL2jBWjl5q6cwTy2RzIl3eD5Sc23iZGVBLP7yKh3V8foq83JUv6Ay4Qje659a6y/5k7mMsNbKnEZq/+hKWOeD2IS7FgOGX/Skbv4pgF0sAVQLv6gx/eacGnei6ZzmINoDE/3kUFhpiIN3veLoufaTiQYabptwI+KelZx0gqJ5hYoNCWx2Jhdrq61pr7f0UKP24ghfMC/OKsavZ6A7Pn+cFf4vV+24YSyovUTG1my7kwhHzgvvjnHNY79UQZ3andrLFcncPomm+N85ATmsEWDpC2pS7w5mdpdmgeieJ5sVmyqVuef07tCGj0wYmVHM0Twifhg7fVKyWHozN263hJ0xhP9yoqJfxNbgBI3QJjPHwVI0N94+eBmDnvE6zU+N0kHuSiRLwq542pFaYR2FyumUiNfRlkA/XrtCQf27iWdG9cHUOREjhh71+uIzZkwt8TC7Zb32Xj6Djt7lpEZOAtOSaIwQWVOmvx3wa1Q+EIamYPifxqDahjcuw0cNor0fbIg2hkuYyFETbmPVZMDLjno0a2uNayiHRCrgM/67oPs5CzmD4xt4q+cadDaciaHnBMciqC5dtDXhSORx10v26z1A13lHm/rkHrRYGifG5p50rA+LQvPKyU+JGUpC85TQZBZvA/y1mGxA6OmC+Nd/LJ/hSGAXLRabslRqVde+Ts0iTOJBa+MdnXAhHrr6QOYCtD1RLZ2oFKeGdJnZDSUIDmjEo4wvNmvAqX3uPPWO3gVgOefcxBx9BeZn0v007G+psBZ4kWvg4faAF0Ei47DJpPg2GBdswrJqKNYrk6d70T/6FzjxmVBzNv9F0jwqCqICvWV1FzAVXwALrtN7pAA/LBcSzbKXmqSr/hoeD/bfkMs4UlaIPcP8RtMfwrMnYJE1buhWPfkKf/iHhz1I2eofueXZWXa5enymQoJN79eNSEh1PUdqlDVZnSLRFvDrmNsQYM2DjqxTQvmWk8x6kaNu7zaD0/bughOoOT9wB3fg/pPRV4Sp/oDN2uKhv3hYEL4+/LeWFpAT5D4lILcOa54XeTcxIRwQD0/Q/GH6hBlZyCs+n5b2ZeAdgUBU5cw9tcrlWSn0PICF/v/xuQ0kLNVFALwC0bB0rCvlNYeEttW5Ve4R9vwjt0mtw2csjhlW0hHtLn8maVhG1TNB1b2fCzVqHXj/7EnDAx0y91BbN0F4bPBHVd6mwqw2iIQZVwNjkHQf0BShRp53WC8W+tIOXRY+CkXyIocQOmLw9x/ZPnt0S8RGp2p/Qq8kaT8kYn8RRMvwUXwPQabq6ENRlSrWJT3AEjSainLwMg3Vyz8x0XJJTCYGKgGvLmCATik8QqxukjHyC5F3QOBwGQ7xFGVFy0qZMRKTp9wWrNPkgFskPPLvy2BK15WT7kQZbuM1yjEh++P7KxikPR0XXtwcG+jfs4YLYWkEauQSEm7atjKf40hysqJ6jRe2dC043ElpTyp1C25iWc22bGqv0CdPPsxBSN2mXv/nVaKo5Hf6WMrTRc4dcjN2SRrTeKylw26hVEqV6z8+LQ4RPR9zEDkaTvjuIKY13RhLyG5+ERp6bEIIX99mkJ6SPWpJrx55pa1O5GTr8gP3fH/FRfXcAGTxwVCUZbMkSoz7rlXbvsanxZnrhrN4mnDCW6bD0SmUcKOJVWoFvs72WXAgewIg6slo9GFSUTzXcfkZPVGYeBvbSRjGIzelg/I25HyRKAPotZX4GnUhZQQRR7OQtIX48S1uB/dXpFsEk0Zj/zG1Az2TcNeNepdBhJs1j+jNrB2qQgghK3dFFPlhuey/7KaN/x3CecDwZtHg0BCXiPCefK2ZxCXCz3hRO0TV2XcW9YoNAoy+Y19yL1S7Y8rC0i0MyzgTtWTmpkn62xGJ+0jNP9hcQQ7VhM2x6Mu7o0jsO1zQzF5ld95EO0IIGWlZpNp+62NMC9ox+P2LqOZyAvw2jKDCgybAaMAMDmzOu2bBXEP0S26+OqsmjxjdqlVn5xXbPpMy/GzP3VDxfBI6Jneq/3QcOb/0Be12tP8o4APq9LS6IuqtoNCLrGAPqJfyXrZAKFHZ2tdvyPqzSRt/y9JLH1qTODBUr4dIdlRPIaqHadXxqpfUvXJFNq9ABxlIamcd7wc7o49WrKUP17rWWZXtbAbG8L1+xntc284HSgXyvISEZ8kIceK1LNyRvnesCJPP3XZVe9h58EDQgFrmLFiavu3583aYBYUs0M1hl/Wk4yqrRuwDRApL2vYVtRof6fqAb09eXy+epQMbToSh1pl7PlHZVHp5ybIQadNXVNTQYbQFzA2hmnIB9CjTHWDMq8qYZLVR6+eJeyxVmipCejDbmgMZ9ZHLbO1/LdVN2QjxaKoDMNWPgRSkEOXtb6q6E+5uvuvtCl0LI6i6ziE/1RTzl8nB8xkZQG276lvo6adERtCf5MPN07tPIhQcNryl9l5WcT8qYsqF37uIQfvSzUqdQTVn7NDhJcotrT6t6vGpOVR7ZmdyHTYC3dvSe5ED5msOXiOiSA4bfcIWIvqrYLiv/RQsd6TZr5ChGZQgIsin3dwsRlqU9+Ezrgpx4RWfIeEmDIidf+nxplYjTSgtppcUXL5ShEAtWchRU4yZIkl7LVjjVL00WtU1o9px92uihxbsHv9RTGXONLPe6SiGP/6qWBfi5GX7+dvngxprJ5EeryF92Bg4MkweX/cmTfFR3ttS4nd4W92ImLjcaDVl+suqTeCzjTEom3wjaLemWdK5nqqrMhzq8S8SWRBMo726R/T9jloWyfYdeUqnxgkvYKDx+CsH0xXBmiPwZI9qxnsWCgwFzUY3MeZKvv+YpRYopAkxdpsOwB0cpq+QdkQVwp0sROHyX8S9BPpc299nUqgxTZbUIe+uEaDgdjDv1LStexGMEBgSVxV7RSQdaiH/YiGGT5Gp8B2l91Pc5FCVDlsQUzWF++U1ykg0iBQJ8/cHvvVbzvNoIcK1Fgdrkq12keNhml8HSavfl4pgU/sbIB5GvYkKwxOy54s9Ayk2PfRpSwPBSRvw/9rO4RQLBdPHVRzUs8Ledh4Pw3JjtpqL1HNHO00+pzoHOa2aORiHh2CK0tfKpjV17g/6cMMJ8YzLAUf5sQrVUZi4TVQaL8DXB9TOzcGGBwbV8hwTXtFG3gj+X7QCV8H1G5MF2mXSMu+LJHFR3lIeyIgqEyI/4BjxvY0ZacF6ECcMNmSvvWqqaWg5wU6+n7TuuAVkQBuG36y5LkojT//qEs5w6ec5+H37xvqEXBXuksqB3xn12HkRlSWbwjs3HTS8zBPdpFgdtHAI6Op+Hx+XEhXNbmqdkM+2kfuUeFkG0aUDAjty375OTKa3I1wIN0lAsDA1nS61ggWv+UJ/QvFWFlX8r05EVWzAY1uMLRM8awDg/1HmHrW4nO9hJ+aPS2T1LptKKOWJG1j0ZZcR4VIVVW34vH/vca87sTew/9MeBX7rt7X2s5OKqA/RoIm8y61B4ibH7mFqusfvRxcZGZ052bif5h6IOf+JvssK0t572YoUE2pSmH2sH0qlNQzxO5BhMJ0cNMqT+Fx1DdlboFC30rDPFd2qZQUewVsQ9rBkpvfrHwyp8pFbaXHb8gbH6mMt/Za0zh440MUmLEbzhMhPj7dYSFaWmM5HqV6XRoQm9Jg5X7ZZ+04W2I5/dliwfh/2QpLEi3YP955LvBmwXhEk8ji3ThEvCLnuaos4lB+Jd2/rEe1QrWPGyC7dQyoS0HH8CJuVXahWo5Y5XwS/QSOUv6qR8ALnsmAQ4da0r8S3nsfONefF+C/2ZY3D72TiPPpYLlPh0gNG8wWzaABQs7gDrtwqyOygDTOiCLu1DKcADDTKEeQYJGd2cIQOtEpX6aBT2ftfX+QjcXa0RRa0PiUA6Qavza7Pyi82nke5zlyhIPGvC3KqjRtmt+Sy2gQjhC/aHQPM/AmQ3YjwFY7/3qgIhDQsf2S6MJosLzJH/EQzN4IosusJVgmXdIxnMx4GjjcC/infguy+YgkdpUkav3fKuKTntEH/2xPC+BiC/HZ0h7fT0n4G/cMLAFQVzHyO8utqsuvC63NcpMXGirgg+4ZFE0w7gP/uHm7x+2Pq16E18QHfuGCEkfVu/nNfGYaD2ZxlusAFifPUSaEuIwPDi2VXvL0Ube3T7jU5k09HRFufm5oAftTB1L9y/auw90KDQD7CG4YwiEaflQOUZB2qjOEz93P/zNzKu9mpE2o9QNzl/5RtczyalFLJUFNnpEpc/FVby2DB4rWW/4MhseS6QqeFJ4MmL6fYGbe75U3fo+5hjEUqHDWFs99mD0o3mmGDsXUJNg9ajN3zYeIe/dqz3A59iQbGWvF43QmkRgiGYQ5rUuHjZ7b9fnvRxoW+Gcio1rioljzAsEdOHzvjOsz/587s6B9bIaQpsj/Z1EeTxCTOC/51OruGF9am/qPA6DDYn4HzPxcWdqcgAVREXyezPBnHGCjtgn0S7scs2edOLcAPM4cnyz0wq8SMCF1cMPH/MooVhcws5/Roem2uVyIdXq2ZI/ifkGBH75gTq0YaBVKjc5K7g41MuQ07/WeElFp2+ISPxyvrTZqW9JQMRInm5JvlfgwMte9w5CxD/MtdszeIVKGFGdQT6xRaXbrD9YXeWzK1cS5gHJ0JjVGuCvQNPdun95bQFqNwgi2/Pu9wpeVm/5qquffsFuIZEhW61H1+VD6FzdwQo6yJfP00HCkobx1agpOgiIbXDsRK6kVDFDNhuITrLCphoWJA+rVXYL7+FvNYTDZHdS/dvd133HbEdlGk61X1FcAgdBBrAGuQvBvt6+SwHxq3CAq9qbWaH4lwjf86FIfrk8dykBnFkyPFYTJYl9qu4dOThEFgx/jlAn+xQRkhGUjZfloYU0+ICbP9NeG8H83WyKEbz27HIoSpAYjTF+7nD0+kUWQbl2G7owpP7safMIDc3N/8Nh08mXmpn8c89/ORFGyg3RhlLpeQpows126U+f9tBSsERdEXaB1pdcIK3DbYh/1SsuX9eYMV6b0so7JDoExVz82pm4gCd+iz0moa97vnX/4NLbYkD+q9KGIgUXz4Sr1752Sjx6KIQOJe1sb88nmp4Pd5xlMk0cpvCdffMGgV3FXngOUHHKl/vuCxG/7DmelD2qEnAI46axQfKZWGPgghWU5n5TsT8k/nGFMRQ/zSxUkj4NNHU1TV5IxSSLYpwDzMXGPSpMvfUuJu69io0OOKoaAnLuvm1upAI4AAQyOabcHkBVnLyV8J8rE1gaZjYBiP3YY0ySXsbguT9ev6xt9xJRIfQHMMBim0JbCh1ijhigzyaoAyhg12+TlRG1/bHAgMPVR8hurs6DGLFg6d5WM1wsi6n7fW5dwL8EjG/Y7UopXecy6A1x5JARnUPM9JmrKdrIjtABQjUC6egd7EeyqfByACnAFqwLw5m/A3/xie8SchtexWUEkKppTX5GsCBTdBoMjRTTDuawuU9P779WPJF+4e+wrpKZCxz9iVNmMSGZQjwLZFeoCsP0ZNe8SmFRjthV/J7uiHLEIWhK8tv2OL2i7vI/Oe6o/V1OFj2blDbfHsee2pXhrL24atlEKLaC7eImuoZ8vIx2NKayjsqspqia1cRvly+rGyn3ckOWN5KYh7x/DooJd2V6Kizvd+ti3O/ox7zmu2RhyRUFZvfjfG3dz2DW1zZ0QRtF3DonpxbmPauqzZi0NQXTxuaBdRNvRd6Yx8lG1puarF/49TfmKqXn/5n0rrEczTND0z9d38VJzi2vxIRbuNUkGfJn7mKAR3Nk4feCFB+yHcMIHQpS5nj5CmWti5j5qpNHQrlO/nvZAjFB+qZp7P1DgljXL+K56uENyMnim7Y42cVjOrVC+ZEvzw6s="}`

**Comment by isidorn** [comment]
@sekhuat thanks for the great repro step.
The issue was that the LinkDetector was trying to regex match this huge string for links. I added a limit for how long a string has to be in order for the link deteciton to be considered. This fixes the issue

**Comment by sekhuat** [comment]
@isidorn  is this bug fix released?

**Comment by isidorn** [comment]
@sekhuat no, it will be release in around 10 days.
